### DISCUSSION TITLE
⛔ 🛟 Support Request: portfolio template error

### DISCUSSION THREAD

**Discussion by thaandcat** [elementor/elementor#15974] [Developers Community]
I created a new page and used a block titled portfolio. (from template section)
![2021-08-13_17-56-46](https://user-images.githubusercontent.com/59549612/129349294-a9f5dece-729e-40da-8afa-fb91b48a6440.jpg)

and choose to use block according to attachment 
![2021-08-13_17-57-17](https://user-images.githubusercontent.com/59549612/129349346-d5ba5b44-d3b7-4f02-96f6-91099559931a.jpg)

 but it appears that it doesn't display as selected.
![2021-08-13_17-58-03](https://user-images.githubusercontent.com/59549612/129349381-b17f89b3-baa4-4c90-b373-0c0258edccb8.jpg)

plug in latest update works on latest worlpress 5.8
![2021-08-13_18-11-07](https://user-images.githubusercontent.com/59549612/129349409-2cc1a12b-89fb-494d-9e7a-13766becc26c.jpg)

**Comment by nicholaszein** [comment]
Hello @thaandcat ,

Thanks for contacting us! 🙏 

### 🛟 This is more of a support question, not really something we can help with here. 

👉 Please remember to check the [Help Center](https://elementor.com/help) or contact [Elementor Support](https://elementor.com/contact) before submitting an issue.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

### ⚠️ As a gentle reminder, **🛟 Support Requests** may need to be closed without troubleshooting. 

Thank you for understanding! 🙏 

Kind regards
